test/reacli.test.js

```diff
@@ -0,0 +1,146 @@
+		
+			withFixture(testDone, fixture, (done) => {
+				common(["component", componentPath], {
+					cwd: fixturePath,
+				}, (err, code, stdout, stderr) => {
+					const { same } = dircompare.compareSync(expectedPath, componentPath);
+					
+					expect(same).toBeTruthy()
+					expect(code).toEqual(0)
+					done()
+				})
+			})
+		})
+		
+		test("reacli component ./flow-component --flow", (testDone) => {
+			const fixture = new Tacks(Dir())
+			const componentName = "flow-component"
+			const expectedPath = path.resolve(__dirname, "fixtures", componentName)
+			const componentPath = path.resolve(`${fixturePath}/${componentName}`)
+		
+			withFixture(testDone, fixture, (done) => {
+				common(["component", componentPath, "--flow"], {
+					cwd: fixturePath,
+				}, (err, code, stdout, stderr) => {
+					const { same } = dircompare.compareSync(expectedPath, componentPath);
+					
+					expect(same).toBeTruthy()
+					expect(code).toEqual(0)
+					done()
+				})
+			})
+		})
+	
+		test("reacli component ./redux-component --redux", (testDone) => {
+			const fixture = new Tacks(Dir())
+			const componentName = "redux-component"
+			const expectedPath = path.resolve(__dirname, "fixtures", componentName)
+			const componentPath = path.resolve(`${fixturePath}/${componentName}`)
+		
+			withFixture(testDone, fixture, (done) => {
+				common(["component", componentPath, "--redux"], {
+					cwd: fixturePath,
+				}, (err, code, stdout, stderr) => {
+					const { same } = dircompare.compareSync(expectedPath, componentPath);
+					
+					expect(same).toBeTruthy()
+					expect(code).toEqual(0)
+					done()
+				})
+			})
+		})
+	
+		test("reacli component ./scss-component --scss", (testDone) => {
+			const fixture = new Tacks(Dir())
+			const componentName = "scss-component"
+			const expectedPath = path.resolve(__dirname, "fixtures", componentName)
+			const componentPath = path.resolve(`${fixturePath}/${componentName}`)
+		
+			withFixture(testDone, fixture, (done) => {
+				common(["component", componentPath, "--scss"], {
+					cwd: fixturePath,
+				}, (err, code, stdout, stderr) => {
+					const { same } = dircompare.compareSync(expectedPath, componentPath);
+					
+					expect(same).toBeTruthy()
+					expect(code).toEqual(0)
+					done()
+				})
+			})
+		})
+	})
+	
+	describe("Hook creation with CLI", () => {
+		test("reacli hook ./simple-hook", (testDone) => {
+			const fixture = new Tacks(Dir())
+			const hookName = "simple-hook"
+			const expectedPath = path.resolve(__dirname, "fixtures", hookName)
+			const hookPath = path.resolve(`${fixturePath}/${hookName}`)
+		
+			withFixture(testDone, fixture, (done) => {
+				common(["hook", hookPath], {
+					cwd: fixturePath,
+				}, (err, code, stdout, stderr) => {
+					const { same } = dircompare.compareSync(expectedPath, hookPath);
+					
+					expect(same).toBeTruthy()
+					expect(code).toEqual(0)
+					done()
+				})
+			})
+		})
+		
+		test("reacli hook ./scss-hook", (testDone) => {
```

Comment:
Shouldn't it be `"reacli hook ./scss-hook --scss"` instead ? 😊

Reply:
ac5fef5